packages/server/postgres/migrations/postDeploy/1619383917863_updateDeletedUserEmails.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate';
+import updateUser from '../../queries/updateUser'
+import getRethink from '../../../database/rethinkDriver'
+import User from '../../../database/types/User'
+import {backupUserQuery, IBackupUserQueryParams} from '../../queries/generated/backupUserQuery'
+import getPg from '../../getPg'
+import catchAndLog from '../../utils/catchAndLog'
+
+export const shorthands: ColumnDefinitions | undefined = undefined;
+
+const undefinedUserFieldsAndTheirDefaultPgValues = {
+  newFeatureId: null,
+  overLimitCopy: null,
+  isRemoved: false,
+  segmentId: null,
+  reasonRemoved: null,
+  rol: null,
+  // app doesn't allow following fields to be undefined, but found bad data anyway
+  inactive: false,
+  payLaterClickCount: 0,
+  featureFlags: []
+}
+
+const cleanUsers = (users: User[]): IBackupUserQueryParams['users'] => {
+  const cleanedUsers = [] as any
```

Comment:
nit: when the input & output arrays are the same size, try to use `map` to get free type checking & express that it's (probably) a pure function.

e.g. the function could be replaced with:

```ts
const skippedPgUsers =  skippedRethinkUsers.map((user) => ({
  ...undefinedUserFieldsAndTheirDefaultPgValues,
  ...user
}))

Reply:
oooh that's a good catch. i think i was going back and forth on the implementation of the mapping fn, but since this one returns equal length it is a lot more expressive to use map